public/index.css

```diff
@@ -439,56 +511,69 @@ footer img {
     background-color: #141a20;
 }
 
-.large-counter h1 {
-    font-size: 9vh
```

Comment:
Avoid using viewport units on font sizes, as they can become way too small for it to be readable on narrow window sizes.